src/components/shared/LibraryTypeSelect.vue

```diff
@@ -52,19 +47,18 @@ export default {
       type: Boolean,
       default: true,
     },
-    labelCols: {
-      type: Number,
-      default: 2,
-    },
-    labelAlign: {
-      type: String,
-      default: 'left',
-    },
-    labelPosition: {
-      type: [Number, String],
-      default: () => POSITION.Left,
```

Comment:
I've removed a lot of the customisation here. Ultimately I think this should be like the DataTypes select, and be solely concerned with the Select element itself.